### DISCUSSION TITLE
Stripe Module

### DISCUSSION THREAD

**Discussion by ahabalikhan** [brocoders/nestjs-boilerplate#1255] [Ideas]
I've used this boilerplate in one of my projects, one thing I missed was having a payment management system more specifically a stripe module with basic functionalities such as:

- A service containing basic stripe functionalities such as Customer CRUD, Ephemeral Key, Payment Intent CRUD, Create Charge, etc.
- Connect account and Account event webhooks.

Would you like this to be added to the boilerplate?

**Comment by Shchepotin** [comment]
@ahabalikhan  Currently, We are trying to figure out how to include some features during the initialization phase of the project and we have plans to add Stripe once we solve it, because not all projects use Stripe.

**Reply by ahabalikhan** [reply]
@Shchepotin Yeah that makes sense, later on, whenever you want Stripe added just let me know, and I can open a PR for it.

**Reply by Shchepotin** [reply]
@ahabalikhan sure, thanks!

**Reply by cdx111** [reply]
ok
